test/gtest/error_handling.cpp

```
@@ -25,6 +25,8 @@ namespace gtest {
 namespace nixl {
     constexpr const char* ucx_err_handling_mode_key  = "ucx_error_handling_mode";
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)